base/dict.jl

```diff
@@ -69,11 +69,61 @@ Dict{String,Int64} with 2 entries:
 Alternatively, a sequence of pair arguments may be passed.
 
 ```jldoctest
```

Comment:
To fix part of the [failing doctests](https://build.julialang.org/#/builders/148/builds/3366/steps/3/logs/stdio), you could add [a label to the code blocks](https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Preserving-Definitions-Between-Blocks-1) that should share their context:

```suggestion
```jldoctest dictexamples
```